### PR TITLE
Feature/feature global backend script

### PR DESCRIPTION
**Pull request by nicoxx**

### CODE REVIEW

File: src/functests/test_launch_scrap_and_learn.py [hunk 1]
```diff
@@ -16,18 +16,19 @@ def setUp(self):
...
         nb_topics = 500
-        model = TopicModelDescription.make_from_scratch(model_id, [[]])
+        model = TopicModelDescription.make_from_scratch(model_id, [[('w' + str(i), 1)] for i in range(nb_topics)])
         self.dal.topic_model.save(model)
         ref_feature_set_id = 'LaunchScrapAndLearnTests_ref_feature_set_id'
         self.dal.feature_set.save_feature_set(FeatureSet.make_from_scratch(
             ref_feature_set_id, ['feature_' + str(i) for i in range(nb_topics)], model_id))
         self.dal.feature_set.save_ref_feature_set_id(ref_feature_set_id)
-        self.is_coverage = bool(os.environ.get('COVERAGE', None))
+        self.is_coverage = True  #  bool(os.environ.get('COVERAGE', None))
```
Comment (m-mokhtari) [inline src/functests/test_launch_scrap_and_learn.py]: Why forced to True ?

File: src/server/tests/test_structinit.py [hunk 2]
```diff
@@ -33,8 +35,31 @@ def test_create_user_in_db(self):
...
+        initializer = _ProfileInitializer(ref_feature_set_id, model_description)
+        interests = ['t1_w2, t2_w1.', ' useless_word']
+        profile = initializer.get_new_profile(interests)
+        model_data = profile.model_data
+        feature_vec = profile.feature_vector
+        explicit_vec = model_data.explicit_feedback_vector
+        self.assertEquals(0, model_data.positive_feedback_sum_coeff)
+        self.assertEquals(0, model_data.negative_feedback_sum_coeff)
+        self.assertEquals([0, 0], model_data.positive_feedback_vector)
+        self.assertEquals([0, 0], model_data.positive_feedback_vector)
```
Comment (m-mokhtari) [inline src/server/tests/test_structinit.py]: Should be negative_feedback_vector.

File: src/functests/test_launch_scrap_and_learn.py (hunk 1 above)
Comment (nicoxx) [inline src/functests/test_launch_scrap_and_learn.py]: a mistake, already fixed in following commit

File: src/server/tests/test_structinit.py (hunk 2 above)
Comment (nicoxx) [inline src/server/tests/test_structinit.py]: Done

**Pull request merged**